modules/deploy/partials/cloud/tiers.adoc

```diff
@@ -0,0 +1,568 @@
+ifdef::env-byoc[]
+== BYOC tier limits
+
+When you create a BYOC cluster, you select a throughput tier. The following table lists the current cluster-level AWS and GCP tier limits. Legacy tiers may have different limits. For more information, contact https://support.redpanda.com/hc/en-us/requests/new[support^]. 
+
+|=== 
+| Tier | Maximum ingress | Maximum egress | Maximum logical partitions | Maximum connections
+
+| Tier 1 | 20 MBps | 60 MBps | 1,000 | 1,500
+| Tier 2 | 50 MBps | 150 MBps | 2,800 | 3,750
+| Tier 3 | 100 MBps | 200 MBps | 5,600 | 7,500
+| Tier 4 | 200 MBps | 400 MBps | 11,300 | 15,000
+| Tier 5 | 400 MBps | 800 MBps | 22,800 | 30,000
+| Tier 6 | 800 MBps | 1,600 MBps | 45,000 | 60,000
+| Tier 7 | 1,200 MBps | 2,400 MBps | 67,500 | 90,000
+| Tier 8 | 1,600 MBps | 3,200 MBps | 90,000 | 120,000
+| Tier 9 | 2,000 MBps | 4,000 MBps | 112,500 | 150,000
+|===
+
+== BYOC supported regions
+
+[tabs]
+====
+Google Cloud Platform (GCP)::
++
+--
+|=== 
+| Region | Zones | Available throughput tiers
+| asia-south1 
+a| * asia-south1-a 
+* asia-south1-b 
+* asia-south1-c 
+a| * tier-1-gcp-v2-x86
+* tier-2-gcp-v2-x86
```

Comment:
So this region does not get Tiers 8 & 9?

Reply:
correct, not today